docs/integrations.md

```diff
@@ -229,6 +229,17 @@ docker run -v $(pwd):/src --workdir /src returntocorp/semgrep-agent:v1 python -m
     which can be changed to any value
     that `semgrep` itself understands.
 
+
+# Semgrep as an engine
+
+Many other tools use Semgrep inside, but are maintained by others outside r2c!
+
+* [NodeJSScan](https://github.com/ajinabraham/nodejsscan)
```

Comment:
I would like to correct the usage as `nodejsscan`

Reply:
```suggestion
* [nodejsscan](https://github.com/ajinabraham/nodejsscan)
```

Reply:
Thanks @ajinabraham -- fixed!